.github/workflows/build.yml

```diff
@@ -8,8 +12,6 @@ on:
     tags:
       - 'v*'
   pull_request:
-    branches:
```

Comment:
I guess we don't need it for tags as well?

Reply:
You mean `push.branches`? If so we still need it because we define `push.tags`, otherwise build will only be triggered on push tags event.

For `pull_request`, there is no sub field anymore and therefore will trigger against whatever head branch.

Reply:
No, meant the wildcard under tags 😁

See I commented on the wrong line

Reply:
Current release strategy implies a semver tag to be pushed. We want to avoid pushing on Hub and GitHub Release if tag is "malformed".